frontend/src/components/common/QuestionCard.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import {
+  Box,
+  Container,
+  Flex,
+  Image,
+  Spacer,
+  HStack,
+  Tag,
+  Text,
+} from "@chakra-ui/react";
+
+type QuestionCardProps = {
+  date: Date;
+  questionTitle: string;
+  image: string;
+  text: string;
+  tags: string[];
+};
+
+const QuestionCard = ({
+  date,
+  questionTitle,
+  image,
+  text,
+  tags,
+}: QuestionCardProps): React.ReactElement => {
+  return (
+    <Container borderRadius="22px" padding="24px" maxWidth="xl">
+      <Flex>
+        <Text fontSize="16px" lineHeight="21px" align="left">
```

Comment:
for all text, you can use the corresponding textStyle (from `frontend/src/themes/typography.ts`) - it's a little broken rn lol but will be fixed once i merge my [PR](https://github.com/uwblueprint/jump-math/pull/55/files). also double check that the font color is right (should be `grey.300` i think)

```suggestion
        <Text textStyle="eyebrow" align="left">
```